### PR TITLE
Editorial updates to diagnostic topics

### PR DESCRIPTION
**Pull request by SueChaplain**
- fix markdown markup issues
- Add improved links between topics

Signed-off-by: SueChaplain <Sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Review by doveye — CHANGES_REQUESTED** [review]
Just one thing that needs fixing.

File: docs/tool_jdmpview.md [hunk 1]
```diff
@@ -248,21 +248,23 @@ Pattern rules:
...
 - Use the `grep` command to show lines that do not match the pattern.
 
-For example, the following command displays the number of instances and total heap size for the `java/lang/String` class:
+    For example, the following command displays the number of instances and total heap size for the `java/lang/String` class:
 
     > info class | grep java/lang/String              
     94 7688 [Ljava/lang/String;            
     1822 58304 java/lang/String               
     1 16 java/lang/String$CaseInsensitiveComparator               
     0 0 java/lang/String$UnsafeHelpers
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/tool_jdmpview.md:268]: Hmm ... that'll be the `>` ... I didn't spot that for some reason. Trying something else to reformat.

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

File: docs/tool_jdmpview.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/tool_jdmpview.md:267]: With the `>` in there I could not get the build to understand that this was a code block, so I pulled out the `info` command into the preceding sentence and now the rest shows as a code block ok. Think this works.

[the remark this answers is not in the document]
Comment (doveye) [reply docs/tool_jdmpview.md:267]: Unfortunately the 4 lines of the first example are now wrapped together. Did the 3-apostrophes thing not work?

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by AdamBrousseau** [comment]
jenkins doc stage

**Comment by AdamBrousseau** [comment]
jenkins build container

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Pull request merged**